public/components/app.tsx

```diff
@@ -38,6 +38,7 @@ import { PLUGIN_ID, PLUGIN_NAME } from '../../common';
 import { CreateReport } from './create_report/create_report';
 import { Main } from './main/main';
 import { ReportDetails } from './main/report_details/report_details';
+import { ReportDefinitionDetails } from './main/report_definition_details/report_definition_details';
```

Comment:
create a index.ts for component use that to reimport and export 

index.ts
```
export {ReportDefinitionDetails} from './report_definition_details';
```

so other component could import easily
app.tsx
```
import {ReportDenifitionDetails} from './main/report_definition_details';
```

Reply:
Will do